src/app/join/components/Participant/JoinInfoStep/RadioButtonGroupContainer/RadioButtonGroupContainer.tsx

```
@@ -1,25 +1,30 @@
+
 import RadioButtonGroup from './RadioButtonGroup/RadioButtonGroup';
 import {
   labelWrapper,
   radioButtonGroupContainerLayout,
   requiredStar,
   tipWrapper,
 } from './RadioButtonGroupContainer.styles';
+import { errorMessage } from '../../../JoinInput/JoinInput.styles';
 
 interface RadioButtonGroupProps<T> {
+  control: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)